packages/page-accounts/src/Sidebar/Identity.tsx

```diff
@@ -92,6 +66,9 @@ function Identity ({ address, identity }: Props): React.ReactElement<Props> | nu
             subtitle={identity.legal}
             title={identity.display}
           />
+          <Judgements
+            address={address}
+          />
```

Comment:
```suggestion
          <Judgements address={address} />
```

Weird, could have sworn the linter would pick this up (single attribute, multiple lines)

Reply:
✅